testnet-contracts/contracts/BridgeBank/EthereumBank.sol

```diff
@@ -77,6 +86,25 @@ contract EthereumBank {
         return lockedFunds[lockedTokenList[_symbol]];
     }
 
+    /*
+     * @dev: Creates a new Ethereum deposit with a unique id.
+     *
+     * @param _sender: The sender's ethereum address.
+     * @param _recipient: The intended recipient's cosmos address.
+     * @param _token: The currency type, either erc20 or ethereum.
+     * @param _amount: The amount of erc20 tokens/ ethereum (in wei) to be itemized.
+     */
+    function burnFunds(
```

Comment:
should we also be burning the actual token, ie, reducing the supply of the token on the token contract? We would need a BurnableToken kinda ERC20 for the token too then right?

Reply:
Updated to BurnableToken https://github.com/cosmos/peggy/pull/190/commits/a1f617d32b05edf2d1e0db9130c6ee09e7ae14e9